src/ModRamScb.f90

```diff
@@ -246,14 +252,6 @@ SUBROUTINE computehI(iter)
        call system_clock(time1,clock_rate,clock_max)
        starttime=time1/real(clock_rate,dp)
 
-       ! If using SWMF without SCB calculation just take directly from grids
```

Comment:
Question: Why was this removed here? It's obviously not comment-related...

Reply:
It probably shouldn't have been removed in this pull request, but it wasn't actually possible to satisfy that particular if statement.